README.md

```diff
@@ -15,6 +15,12 @@ a Geographic Information System used for geospatial data management and
 analysis, image processing, graphics/map production, spatial modeling, and
 visualization.
 
+## Jupyter Notebook
```

Comment:
I think it would be good to delete the heading. The paragraph/sentence will become just second paragraph/sentence in the document and as a "call to action" it is quite fitting at the end of an intro section.